Includes/menu.hpp

```diff
@@ -80,6 +88,8 @@ class Menu {
 
   void up();
   void down();
+  void pageup();
+  void pagedown();
```

Comment:
Please consider using lowerCamelCase for methods.

Reply:
Done.